Source/Shaders/GlobeFS.glsl

```diff
@@ -67,6 +71,7 @@ varying vec3 v_mieColor;
 vec4 sampleAndBlend(
     vec4 previousColor,
     sampler2D texture,
+    sampler2D textureColorPalette,
```

Comment:
Or you can pass the index directly [here](https://github.com/AnalyticalGraphicsInc/cesium/pull/4953/files#diff-772d885056ceb2379c95a5dfc07d8d5aR190)